third_party/md5/md5.c

```diff
@@ -238,11 +236,11 @@ void MD5_Update(MD5_CTX *ctx, const void *data, unsigned long size)
 		memcpy(&ctx->buffer[used], data, available);
 		data = (const unsigned char *)data + available;
 		size -= available;
-		body(ctx, ctx->buffer, 64);
+		LXW_body(ctx, ctx->buffer, 64);
```

Comment:
P.S., Is renaming body() to LXW_body necessary? It isn't exported as far as I can see.

Reply:
Not necessary - just did it for consistency.